Makefile

```diff
@@ -88,8 +88,7 @@ qemu: selfie.m selfie.s
 # Test boolector SMT solver
 boolector: smt
 	boolector manuscript/code/symbolic/simple-assignment.smt -e 0 > selfie_boolector.sat
-	[ $$(grep ^sat$$ selfie_boolector.sat | wc -l) -eq 2 ]
-	[ $$(grep ^unsat$$ selfie_boolector.sat | wc -l) -eq 1 ]
```

Comment:
The unsat case is still possible, right? Why not check for that?

Reply:
The last if statement is also merged in the example, which means that only one path remains at the end, and thus only one formula remains to be solved, which is satisfiable in this case, so the unsat case no longer exists.

Reply:
Got it!